### PR TITLE
CQ-4346477 Added repeatability support, runtime tests including repeatability, updated authouring tests.

### PR DESCRIPTION
**Pull request by sakshi-arora1**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

**Comment by sakshi-arora1** [comment]
Screenshot 2022-09-13 at 12 17 25 AM
Screenshot 2022-09-13 at 12 17 47 AM
Screenshot 2022-09-13 at 12 18 08 AM
Screenshot 2022-09-13 at 12 18 25 AM

File: ui.tests/test-module/specs/panelcontainer/panelcontainer.runtime.spec.js [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+            formContainer = p;
+        })
+    });
+
+    const checkView = (id, view, count) => {
+        expect(view.children.length, "panel has children equal to count").to.equal(count);
+    }
+
+    const checkViewUpdate = (id, view, count) => {
+        //adding delay for view update via mutation observer after html is appended
```
Comment (sakshi-arora1) [inline ui.tests/test-module/specs/panelcontainer/panelcontainer.runtime.spec.js:34]: @rismehta This test pass in browser mode and fails in headless. cy.wait isn't effective. Any pointers how to handle such mutation observer based tests in headless.

File: ui.frontend/src/view/FormPanel.js [hunk 2]
```diff
@@ -16,16 +16,170 @@
...
+                let beforeElement = this.children[instanceId - 1].element.parentElement;
+                beforeElement.after(htmlElement);
+            }
+        } else {
+            console.error('Add child instance is only supported for repeatable panels.');
+        }
+    }
+
+    removeChildInstance(removedModel, state) {
+        if (this.repeatable) {
```
Comment (nit23uec) [inline ui.frontend/src/view/FormPanel.js:146]: `state` is unused arg

Comment (sakshi-arora1) [reply ui.frontend/src/view/FormPanel.js:146]: removed, unused state

**Review by nit23uec — APPROVED** [review]

**Pull request merged**